src/pages/fresh_install/FreshInstallPage.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import '../fresh_install/FreshInstallPage.css';
+import path4 from "../../images/path4.svg";
+import classNames from 'classnames';
+
+export default class FreshInstallPage extends React.Component<{}, { selectedTemperatureScale: string }> {
+
+    handleButtonClick = (selectedTemperatureScale: 'C' | 'F') => {
+        if (this.state === null || !(this.state.selectedTemperatureScale === selectedTemperatureScale)) {
```

Comment:
Every state should be initialized on the constructor:

```js
class FreshInstallPage extends React.Component {
    constructor(props: Props) {
        super(props);
        this.state = { selectedTemperatureScale: null };
    }
}
```

Reply:
👍
Also, the constructor method is not necessary if you want to just initialize the state.

You could do something like.

```
class FreshInstallPage extends React.Component {
        state = { selectedTemperatureScale: null };
}
```